overlord/snapstate/check_snap.go

```diff
@@ -449,20 +571,9 @@ func earlyEpochCheck(info *snap.Info, snapst *SnapState) error {
 }
 
 // check that the listed system users are valid
-var (
-	findUid = osutil.FindUid
-	findGid = osutil.FindGid
-)
-
-// TODO: keep this unsupported until it is complete!
-var systemUsernamesSupported = false
+var ensureUserGroup = osutil.EnsureUserGroup
```

Comment:
(nitpick^99) its slightly more common to use `osutilEnsureUserGroup` here but hardly a blocker.

Reply:
Done